src/pages/steps/AdditionalInformationStep/BookingInfoStep.tsx

```diff
@@ -485,9 +485,7 @@ const BookingInfoStep = ({
           as="form"
           width="45%"
           spacing={4}
-          onBlur={handleSubmit((data) => {
-            handleAddBookingInfoMutation(data);
-          })}
+          onBlur={() => handleAddBookingInfoMutation(getValues())}
```

Comment:
Is the `handleSubmit` not necessary anymore?

Reply:
The issue was that the signatures were incompatible since you can't usually use it on an ù onBlur`, it would still technically work because we don't rely on the event itself but this was bending `handleSubmit`'s intended purpose a bit so I went for a pure `getValues` to please TS + me :D